Helper/Data.php

```diff
@@ -656,4 +668,53 @@ public function clearMagentoCache($type)
             $this->cacheManager->clean($clearTypes);
         }
     }
+
+    /**
+     * @param ModuleDataSetupInterface $setup
+     * @throws LocalizedException
+     * @throws \Zend_Validate_Exception
+     */
+    public function addCustomerReference(ModuleDataSetupInterface $setup)
```

Comment:
Let's not put this into the data helper. I would create a similar Core class for data what we have for schema: https://github.com/Nosto/nosto-magento2/blob/develop/Setup/Core.php#L47. Then extend that in `UpgradeData` and `InstallData`